apidoc

```diff
@@ -50,5 +50,33 @@ $application = new yii\console\Application([
 if ($vendorPath !== null) {
     $application->setVendorPath($vendorPath);
 }
+$bowerDirs = [
+    __DIR__ . '/vendor/bower',
+    __DIR__ . '/vendor/bower-asset',
+    __DIR__ . '../../bower',
+    __DIR__ . '../../bower-asset',
+    __DIR__ . '../../../../bower',
+    __DIR__ . '../../../../bower-asset',
+];
+foreach ($bowerDirs as $dir) {
+    if (file_exists($dir)) {
+        Yii::setAlias('@bower', $dir);
+        break;
+    }
+}
+$npmDirs = [
+    __DIR__ . '/vendor/npm',
+    __DIR__ . '/vendor/npm-asset',
+    __DIR__ . '../../npm',
+    __DIR__ . '../../npm-asset',
+    __DIR__ . '../../../../npm',
+    __DIR__ . '../../../../npm-asset',
+];
+foreach ($npmDirs as $dir) {
+    if (file_exists($dir)) {
+        Yii::setAlias('@npm', $dir);
+        break;
+    }
+}
```

Comment:
I've pushed a new version. What about this, @cebe? Should I use another set of directories? Thanks!

Reply:
looks great, however I just noticed that `$vendorPath` could possibly be null, not sure what the use case is but your code should go into the if statement above that checks for `if ($vendorPath !== null) {`. Its ready to merge after that. Thanks!

Reply:
Ok!

Reply:
Looks good now?